experimental/origin-dapp2/src/pages/App.js

```diff
@@ -83,6 +85,20 @@ class App extends Component {
             <Route path="/notifications" component={Notifications} />
             <Route path="/about/dapp-info" component={DappInfo} />
             <Route path="/about/tokens" component={AboutToken} />
+            <Route
```

Comment:
can we have a `const GrowthEnabled = process.env.ENABLE_GROWTH === 'true'` at the top and then `{GrowthEnabled && <Route path="/about/tokens" component={GrowthCampaigns} />}` here?

Reply:
👍